concordium-consensus/src/Concordium/GlobalState/Persistent/Accounts.hs

```diff
@@ -256,24 +413,67 @@ updateAccountsAtIndex' fupd ai = fmap snd . updateAccountsAtIndex fupd' ai
   where
     fupd' = fmap ((),) . fupd
 
+-- | Get a list of all account addresses and their associated account indices.
+--  There are no guarantees of the order of the list. This is because the resulting list is potentially
+--  a concatenation of two lists of account addresses.
```

Comment:
The last sentence isn't really useful. The diffMap flatten on its own says nothing about the order.